waspc/src/Wasp/Util.hs

```diff
@@ -49,3 +51,13 @@ jsonSet _ _ _ = error "Input JSON must be an object"
 
 indent :: Int -> String -> String
 indent numSpaces = intercalate "\n" . map (replicate numSpaces ' ' ++) . splitOn "\n"
+
+concatShortPrefixAndText :: String -> String -> String
+concatShortPrefixAndText prefix "" = prefix
+concatShortPrefixAndText prefix text =
+  let (l : ls) = lines text
+   in prefix ++ l ++ if null ls then "" else "\n" ++ indent (length prefix) (intercalate "\n" ls)
+
+concatPrefixAndText :: String -> String -> String
```

Comment:
how is this different from the question above (short one)? maybe explain in the comment above when to use it.

Reply:
Added comments for this one also!

```hs
-- | Given a prefix and text, concatenates them in the following manner:
-- - If just one line of text:
-- <prefix> <one_and_only_line_of_text>
-- - If multiple lines of text:
-- <prefix>
--   <text_line_1>
--   <text_line_2>
--       ...
--   <text_line_N>
--
-- __Examples__
--
-- @
-- >>> putStrLn $ concatPrefixAndText "Log messages from the somelog.txt file: " "Written to file foo.txt"
-- Log messages from the somelog.txt file: Written to file foo.txt
-- @
--
-- @
-- >>> putStrLn $ concatPrefixAndText "Log messages from the somelog.txt file:" "Written to file foo.txt\nWritten to file bar.txt"
-- Log messages from the somelog.txt file:
--   Written to file foo.txt
--   Written to file bar.txt
-- @
```